ooniprobe/View/TestResults/Footer/UploadFooterViewController.m

```diff
@@ -90,7 +90,8 @@ -(void)uploadMeasurements:(NSArray *)notUploaded startAt:(int)idx{
         Measurement *currentMeasurement = [notUploaded objectAtIndex:idx];
         dispatch_async(dispatch_get_main_queue(), ^{
             [MBProgressHUD HUDForView:self.navigationController.view].label.text =
-            NSLocalizedFormatString(@"Modal.ResultsNotUploaded.Uploading", [NSString stringWithFormat:@"%d/%ld", idx+1, [notUploaded count]]);
+            NSLocalizedFormatString(@"Modal.ResultsNotUploaded.Uploading",
```

Comment:
Please also indent `NSLocalizedFormatString` four spaces on the right, otherwise it's not apparent that this line is a continuation from the previous line.